resources/views/users/settings/password.blade.php

```diff
@@ -23,13 +23,13 @@
                     @endif
 
                     <div class="col-span-12">
-                        <x-forms.label for="password">New Password</x-label>
+                        <x-forms.label for="password">New Password</x-forms.label>
```

Comment:
Nice spot 😉

Reply:
👌

Reply:
I did note on this that it would seem Blade UI kit is slightly broken because it always uses the fallback for the label.